lib/cisco_node_utils/cmd_ref/encapsulation.yaml

```diff
@@ -0,0 +1,22 @@
+# encapsulation_profile_vni
+# encapsulation_profile_vni feature is available only on n7k
+---
+_exclude: [N3k, N5k, N6k, N9k, ios_xr]
+
+all_encaps:
+  multiple: true
+  get_command: "show running-config vni"
+  get_value: '/^encapsulation profile vni (\S+)/'
+
+create:
+  set_value: "encapsulation profile vni <profile>"
+
+destroy:
+  set_value: "no encapsulation profile vni <profile>"
+
+dot1q_map:
+  get_command: "show encapsulation profile"
+  get_value: '/^[a-zA-Z\s]+ vni <profile>\s+dot1q ([0-9,-]+) vni ([0-9,-]+)/'
```

Comment:
Would be good to have sample output that this is matching as well.  I think that this could be further simplified.